src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -0,0 +1,29 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.mgmt.keyvault.models.key_vault_management_client_enums import (SkuFamily, SkuName)
+from azure.cli.commands.parameters import (
+    get_resource_name_completion_list,
+    get_enum_type_completion_list,
+    name_type)
+from azure.cli.commands import register_cli_argument
+import azure.cli.commands.arm # pylint: disable=unused-import
+
+from azure.cli.command_modules.keyvault._validators import (process_policy_namespace, process_set_policy_perms_namespace)
+
+register_cli_argument('keyvault', 'vault_name', arg_type=name_type, completer=get_resource_name_completion_list('Microsoft.KeyVault/vaults'), id_part='name')
+register_cli_argument('keyvault create', 'vault_name', completer=None)
+
+register_cli_argument('keyvault create', 'sku_family', completer=get_enum_type_completion_list(SkuFamily), choices=[e.value for e in SkuFamily])
```

Comment:
>completer=get_enum_type_completion_list(SkuFamily) [](start = 55, length = 50)

These are not necessary. If you have a choice list, it doubles as the completer. The purpose of "get_enum_type_completion_list" is for those obnoxious "model_as_string=True" swagger parameters where we can't use the enum as a choice list and thus have to expose it as a completer.